pkg/kev/changeset.go

```diff
@@ -174,72 +173,72 @@ func (cset changeset) HasNoPatches() bool {
 	return len(cset.changes()) <= 0
 }
 
-func (cset changeset) applyVersionPatchesIfAny(o *composeOverride, reporter io.Writer) {
+func (cset changeset) applyVersionPatchesIfAny(o *composeOverride) {
 	chg := cset.version
 	if reflect.DeepEqual(chg, change{}) {
 		return
 	}
-	chg.patchVersion(o, reporter)
+	chg.patchVersion(o)
 }
 
-func (cset changeset) applyServicesPatchesIfAny(o *composeOverride, reporter io.Writer) {
+func (cset changeset) applyServicesPatchesIfAny(o *composeOverride) {
 	for _, change := range cset.services {
-		change.patchService(o, reporter)
+		change.patchService(o)
 	}
 }
 
-func (cset changeset) applyVolumesPatchesIfAny(o *composeOverride, reporter io.Writer) {
+func (cset changeset) applyVolumesPatchesIfAny(o *composeOverride) {
 	for _, change := range cset.volumes {
-		change.patchVolume(o, reporter)
+		change.patchVolume(o)
 	}
 }
 
-func (chg change) patchVersion(override *composeOverride, reporter io.Writer) {
+func (chg change) patchVersion(override *composeOverride) {
 	if chg.Type != UPDATE {
 		return
 	}
 	pre := override.Version
 	newValue := chg.Value.(string)
 	override.Version = newValue
-	_, _ = reporter.Write([]byte(fmt.Sprintf(" → version updated, from:[%s] to:[%s]\n", pre, newValue)))
+	log.DebugDetailf("version updated, from:[%s] to:[%s]", pre, newValue)
```

Comment:
Why not just Debug? is DebugDetail different from a regular debug?

Reply:
It was to differentiate between the old use of Debug ... DebugDetail uses a different prefix (` → `) from Debug.

Saying that, I don't think we use the old Debug anymore. If that's the case, then I'll change `DebugDetail` -> `Debug`.

Reply:
done